Tailviewer/BusinessLogic/ActionCenter/OpenFolderException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Tailviewer.BusinessLogic.ActionCenter
+{
+	public sealed class OpenFolderException : Exception

```

Comment:
What problem should this exception describe?
If the folder doesn't exist, then a name such as "DirectoryNotFoundException" is more telling than "OpenFolderException".

Reply:
If the network drive isn't accessible, it is not the same as when the directory does not exist. Consider this, you don't have an access to the shared directory anymore.
In this case, DirectoryNotFoundException is misleading, wouldn't you agree?

Reply:
Agreed, how about throwing a "NetworkDriveNotAccessibleException" exception in that case?